src/routes/edit/share-options/exports/export-modal.tsx

```diff
@@ -143,8 +193,10 @@ export const ExportModal = () => {
 	}
 
 	const jsonCode: any = getFragmentJsonExportString(fragmentExportModal.fragment, fragments, styleClasses);
-	const reactCode: any = createReactApp(fragmentExportModal.fragment, fragments);
-	const angularCode: any = createAngularApp(fragmentExportModal.fragment, fragments);
+
+	const onVersionChange = ({ selectedItem }: any) => {
```

Comment:
Should this just be an inline function in dropdown onChange?

Reply:
Oh I answered this in your previous comment, not sure if I accidentally `resolved` it.

> Yes? No? It's generally not a good practice to pass inline functions as props due to reconciliation. When you pass a new function reference as a prop it can trigger unnecessary re-renders.